### PR TITLE
HDDS-2767. security/SecuringTDE.md

### PR DESCRIPTION
**Pull request by iamabug**
## What changes were proposed in this pull request?

translation to https://hadoop.apache.org/ozone/docs/0.5.0-beta/security/securingtde.html

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2767

## How was this patch tested?

hugo server

### CODE REVIEW

**Comment by iamabug** [comment]
@cxorm @smengcl @xiaoyuyao Please help review this if available, thanks.

File: hadoop-hdds/docs/content/security/SecuringTDE.zh.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: "透明数据加密"
+date: "2019-April-03"
+summary: 透明数据加密（Transparent Data Encryption，TDE）以密文形式在磁盘上保存数据，但可以在用户访问的时候自动进行解密。TDE 以键或桶为单位进行加密。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/security/SecuringTDE.zh.md:4]: Can we remove this "TDE 以键或桶为单位进行加密"? Also please remove the EN part as well.

Comment (iamabug) [reply hadoop-hdds/docs/content/security/SecuringTDE.zh.md]: good call, done.

File: hadoop-hdds/docs/content/security/SecuringTDE.zh.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+参数名 |  值
+-----------------------------------|-----------------------------------------
+hadoop.security.key.provider.path  | KMS uri. <br> 比如 kms://http@kms-host:9600/kms
+
+### 使用 TDE
+如果你的集群已经配置好了 TDE，那么你只需要创建加密密钥并启用桶加密即可。
+
+创建加密密钥的方法为：
+    * 使用 hadoop 密钥命令创建桶加密密钥，和
```
[the remark this answers is not in the document]
Comment (iamabug) [reply hadoop-hdds/docs/content/security/SecuringTDE.zh.md]: Sorry for this careless editing.

**Pull request merged**